floor/floor/util/serial_read.py

```diff
@@ -51,7 +54,7 @@ def synchronize(self):
     @staticmethod
     def is_stop_marker(val):
         # The first 6 bits of the first byte should always be zero for normal data so this condition is unique
-        return val[0] == chr(0xFF) and val[1] == chr(0xFF)
+        return ord(val[0]) == 0xFF and ord(val[1]) == 0xFF
```

Comment:
Fucking python converting everything to unicode.  Make it all be numbers instead